pkg/cluster/kubernetes/resource/load_test.go

```diff
@@ -319,9 +364,24 @@ func TestChartTracker(t *testing.T) {
 	}
 	for _, f := range chartfiles {
 		fq := filepath.Join(dir, f)
-		if m, err := Load(dir, []string{fq}); err != nil || len(m) != 0 {
+		if m, err := Load(dir, []string{fq}, false); err != nil || len(m) != 0 {
 			t.Errorf("%q not ignored as a chart should be", f)
 		}
 	}
 
 }
+
+func TestLoadSomeWithSopsDecryption(t *testing.T) {
```

Comment:
This test adds value (since it makes sure that loading plain manifests works when sops is enabled). However, I am missing a test (by extending this one or adding an additional one) in which an encrypted manifests is decoded successfully.

The easiest would be using GPG. You may want to borrow ideas for scaffolding from the GPG package https://github.com/fluxcd/flux/tree/6a0e353061f68ba902161af3b39b4e6557c23646/pkg/gpg . Particularly https://github.com/fluxcd/flux/blob/6a0e353061f68ba902161af3b39b4e6557c23646/pkg/gpg/gpgtest/gpg.go#L18

Reply:
Thanks. This was my missing piece :)

I was trying to figure out how test an actual decryption